spark/v3.3/build.gradle

```diff
@@ -39,6 +39,7 @@ project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}") {
   }
 
   dependencies {
+    implementation group: 'com.github.davidmoten', name: 'hilbert-curve', version: '0.2.2'
```

Comment:
What is the licensing info on this library? Just want to make sure it is allowed to be included in the project?

Reply:
I checked this while reviewing. It is Apache-2.0 and is compatible with the Iceberg project licensing.
https://github.com/davidmoten/hilbert-curve

Reply:
Interesting. TIL.

I thought only if we keep their source code in our code, we need to add in [LICENCE](https://github.com/apache/iceberg/blob/master/LICENSE)

We do depend on a lot of third-party libraries and I don't see them in LICENSE. Do you know why?

Reply:
Maybe that's ok then? I just want to make sure we do it right.